### DISCUSSION TITLE
laravel 安装是出现 No publishable resources for tag [laravel-assets] 错误

### DISCUSSION THREAD

**Discussion by JamesXia** [w7corp/easywechat#2820] [General]
@php artisan vendor:publish --tag=laravel-assets --ansi --force

   INFO  No publishable resources for tag [laravel-assets].

**Comment by overtrue** [comment]
这不是错误，仅仅表示没有新的文件需要拷贝，再者，这个是 laravel 的官方资源，和这个 SDK 没有任何关联哈
